pom.xml

```diff
@@ -944,8 +944,62 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>2.10.3</version>
+	  <configuration>
+            <additionalparam>${beam.javadoc_opts}</additionalparam>
+            <windowtitle>Apache Beam SDK for Java, version ${project.version} API</windowtitle>
+            <doctitle>Apache Beam SDK for Java, version ${project.version}</doctitle>
+            <overview>${basedir}/${beam-top}/sdks/java/javadoc/overview.html</overview>
+            <use>false</use>
+            <quiet>true</quiet>
+            <bottom><![CDATA[<br>]]></bottom>
+            <offlineLinks>
```

Comment:
per discussion, drop if the `ant` build does this correctly. Rather have only one place to get all this right.